gapic/templates/%namespace/%name_%version/%sub/services/%service/_shared_macros.j2

```diff
@@ -371,3 +371,61 @@ class {{ async_method_name_prefix }}{{ service.name }}RestInterceptor:
     {% endfor %}
     {% endif %}
 {% endmacro %}
+
+{% macro generate_mixin_call_method(service, api, name, sig, is_async) %}
+{% set async_prefix = "async " if is_async else "" %}
+{% set async_method_name_prefix = "Async" if is_async else "" %}
+{% set async_suffix = "_async" if is_async else "" %}
+{% set await_prefix = "await " if is_async else "" %}
+
+@property
+def {{ name|snake_case }}(self):
+    return self._{{ name }}(self._session, self._host, self._interceptor) # type: ignore
+
+class _{{ name }}(_Base{{ service.name }}RestTransport._Base{{name}}, {{ async_method_name_prefix }}{{service.name}}RestStub):
+    {% set body_spec = api.mixin_http_options["{}".format(name)][0].body %}
+    {{ response_method(body_spec) | indent(4) }}
+    
+    {{ async_prefix }}def __call__(self,
+        request: {{ sig.request_type }}, *,
+        retry: OptionalRetry=gapic_v1.method.DEFAULT,
+        timeout: Optional[float]=None,
+        metadata: Sequence[Tuple[str, str]]=(),
+        ) -> {{ sig.response_type }}:
+
+        r"""Call the {{- ' ' -}}
+        {{ (name|snake_case).replace('_',' ')|wrap(
+                width=70, offset=45, indent=8) }}
```

Comment:
It could be difficult to read when this is on a new line. Should we keep it on a single line?

Reply:
addressed.